### PR TITLE
#303 add tests for ContainerCreator

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #303)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-agent/src/test/scala/com/island/ohara/agent/docker/TestContainerCreator.scala [hunk 1]
```diff
@@ -37,18 +34,76 @@ class TestContainerCreator extends SmallTest with Matchers {
...
                                                 volumeMapping: Map[String, String],
                                                 networkDriver: NetworkDriver) => {
-    // we check only the required arguments
-    CommonUtils.requireNonEmpty(imageName)
-    Objects.requireNonNull(ports)
-    Objects.requireNonNull(envs)
-    Objects.requireNonNull(route)
-    Objects.requireNonNull(volumeMapping)
-    Objects.requireNonNull(networkDriver)
+    // do nothing
```
Comment (chia7712) [inline ohara-agent/src/test/scala/com/island/ohara/agent/docker/TestContainerCreator.scala]: You should keep those checks in order to test "good case".

Comment (saivirtue) [reply ohara-agent/src/test/scala/com/island/ohara/agent/docker/TestContainerCreator.scala]: I think the checker is inside the ContainerCreator, was not ?

Comment (chia7712) [reply ohara-agent/src/test/scala/com/island/ohara/agent/docker/TestContainerCreator.scala]: > I think the checker is inside the ContainerCreator, was not ?

But we need a test to make sure ContainerCreator does have all checkers

Comment (chia7712) [reply ohara-agent/src/test/scala/com/island/ohara/agent/docker/TestContainerCreator.scala]: 簡單來說我們需要一個測試來證明抵達doXXXX()的參數都是"檢查過的"

Comment (saivirtue) [reply ohara-agent/src/test/scala/com/island/ohara/agent/docker/TestContainerCreator.scala]: OK...我了解意思了，等等來修改

**Pull request merged**